src/templates/ControlTemplate.jsx

```diff
@@ -2,29 +2,50 @@ import PropTypes from "prop-types";
 import { Accordion, Checkbox, Textarea } from "@trussworks/react-uswds";
 import ProjectHeader from "../molecules/ProjectHeader";
 import ResponsibilityBox from "../atoms/ResponsibilityBox";
+import { isEmpty } from "../utils";
+import Config from "../config";
+import RequestService from "../services/RequestService";
+import { useParams } from "react-router-dom";
+import { useState } from "react";
+import { Navigate } from "react-router-dom";
 
-export default function ControlTemplate({
-  project,
-  control,
-  nextControlId,
-  inheritedComponentNarratives,
-}) {
+const InheritedComponentNarratives = ({ inherited }) => {
+  if (inherited === undefined) {
+    return "";
+  }
+  return (
+    <>
+      {Object.keys(inherited).map((component, i) => (
+        <div key={i}>
+          <b>{component}</b>
+          <p>{inherited[component].description}</p>
+        </div>
+      ))}
+    </>
+  );
+};
+
+export default function ControlTemplate({ project, control, componentData }) {
+  const { controlId } = useParams();
+  const [response, setResponse] = useState(false);
   const { id: projectId, acronym, title } = project;
   const {
-    controlId,
-    controlTitle,
+    label,
+    title: controlTitle,
     description,
     family,
-    guidance,
-    implementationStandards,
-    responsibility,
+    guidance = "No control guidance found for this control",
+    implementation = "No implementation standards found for this control",
     version,
+    next_id: nextControlId,
```

Comment:
If you're changing the variable name from `controlId` to `label`, then we should probably change this line to reflect that as well, maybe something like:
`next_control: nextControlLabel`

Reply:
I can aim for a more consistent name